### PR TITLE
Split chrome script into several helper files

### PR DESCRIPTION
**Pull request by leonbubova**
## Summary
This PR introduces quite a lot of changes. Since `return-youtube-dislike.script.js` has grown quite big, I tried to split it up into several smaller, more maintainable and possibly testable files and created a file structure where I organised the new files thematically, using ES6 modules.

For most of the changes I just moved the functions into their own files and didn't change anything of the functionality itself, other than a handful of changes, but I did introduce some extra parameters to some of the functions to fix the not globally reachable parameters of some functions now.

### What
- Split up `return-youtube-dislike.script.js` into several helper files (see Screenshot for proposed file structure)
- created ES6 modules out of the helper files to make them importable and reusable throughout the project
- refactored usage of RYDTools and RYD class to be more simpler, since they didn't work that good with the new structure

### Why
Big script files can get out of hand soon and become unmaintainable and untestable. Therefore we should address this sooner rather than later.

### New File Structure
image

### CODE REVIEW

**Comment by Anarios** [comment]
Looks very impressive, thanks. Check latest changes in main (FF and chrome versions are combined now)

**Comment by leonbubova** [comment]
> Looks very impressive, thanks. Check latest changes in main (FF and chrome versions are combined now)

Thank you, unfortunately I ran in a lot of issues and probably have to redo a lot of what I already made, but I am optimistic that I can make something work with the help of webpack.

Will have a look at `main` and change accordingly. Sounds nice!

File: Extensions/chrome/return-youtube-dislike.content-script.js [hunk 1]
```diff
@@ -1,10 +1,70 @@
...
+import likeClicked from "./Helpers/State/likeClicked";
+import dislikeClicked from "./Helpers/State/dislikeClicked";
+import { LIKED_STATE, DISLIKED_STATE, NEUTRAL_STATE } from "./Helpers/State/states";
+import cLog from "./Helpers/Log/cLog";
+import numberFormat from "./Helpers/Numbers/numberFormat";
+import roundDown from "./Helpers/Numbers/roundDown"
+import setDislikes from "./Helpers/State/setDislikes";
+import getButtons from "./Helpers/Ui/Buttons/getButtons";
+import getDislikeButton from "./Helpers/Ui/Buttons/getDislikeButton";
+import getLikeButton from "./Helpers/Ui/Buttons/getLikeButton";
```
Comment (glenlalo) [inline Extensions/chrome/return-youtube-dislike.content-script.js:23]: Creating separate files for each function might be an over-kill. Pushing this pattern further means that other contributors have to create also a file per function, and they have to find also the perfect location for the file. With this file structure, searching for functions will become searching for files.
Re-grouping related functions would be better. 
getButtons, getDislikeButton, getLikeButton are all buttons and can be placed in buttons.js. 
likeClicked, dislikeClicked are events and can be placed in events.js

Comment (leonbubova) [reply Extensions/chrome/return-youtube-dislike.content-script.js:23]: Yes, after sleeping on it I do agree with you as it seems pretty heavy to have one file per function. I will redo this PR and re-group related functions in the next iteration of this PR 👍

**Comment by leonbubova** [comment]
Restructured what I did already and created 4 separate src files. Also removed the RYDTools logic and made it simpler. Please wait with reviewing until I updated the PR body though.

**Comment by Anarios** [comment]
Does it need to be built with webpack, or imports work without building?

**Comment by leonbubova** [comment]
Needs to be build, just do `npm build` that creates the bundled-content-script.js

**Comment by Anarios** [comment]
Do you have time to configure webpack in suck a way that we can npm-build --browser=chrome and npm-build --browser=firefox, and when chrome is selected it copies everything and manifestv3 into dist folder (and renames manifest v3 to manifest.js) ? I played with it a bit yesterday, but I suck at webpack configs.

**Comment by Anarios** [comment]
As a separate PR or right here - whichever way is faster for you.

**Comment by leonbubova** [comment]
Hey, that's sound's like a nice idea, I'll create an issue and will have a look (also a webpack noob :D Would love to level up my skills)

I'd say let's do that in a separate PR though, since the webpack implementation is already merged anyways.

**Comment by Anarios** [comment]
@leonbubova I've done that, a bit ugly - but works for now. Will improve it with time.

**Pull request merged**